Week 4/Spotify OOP/artist.php

```diff
@@ -0,0 +1,31 @@
+<?php
+    session_start();
+    require_once("classes/Artist.class.php");
```

Comment:
Hier kan je eventueel een autoloader inschakelen (of via een bootstrap.php werken)